tests/rule_handling_test.py

```diff
@@ -585,56 +585,28 @@ def revert_mount_options_test(self):
         self.assertEqual(self.storage_mock.mountpoints["/tmp"].format.options,
                          "defaults")
 
-    def revert_password_changes_test(self):
+    def revert_password_policy_changes_test(self):
```

Comment:
Same as above - there are tests removed from this method, and I am not really sure it is good approach to remove them when they fail. We should probably check this behaviour in separate PR, if the problem is in test or in the code itself.

Reply:
@dahaic see above. This test needs to be changed to reflect that root password is no longer cleared.

Reply:
ack